management/users/users.js

```diff
@@ -12,77 +12,88 @@ WDN.initializePlugin('modal', [function() {
 	require(['jquery'], function($) {
 		window.jQuery = $;
 		require(['bootstrap-select'], function() {
-			$(function(){
-				// display new user modal
+			$(function() {
+				initializeAddUserModalEventHandlers();
+				initializeUserPermissionsSelectPickerEventHandlers();
+				initializeUserAbilitiesSelectPickerEventHandlers();
+				initializeEditButtonEventHandlers();
+
+				// Initially, create the user table
+				refreshUserTable();
+			});
+
+			function initializeAddUserModalEventHandlers() {
 				$('#add-user').colorbox({
 					inline: true, 
 					width: '50%'
 				});
 
-				$('.close-modal-button').click(function(){
+				$('.close-modal-button').click(function() {
 					$.colorbox.close();
 				});
 
-				$('#add-user-form').on('submit', function(event){
+				$('#add-user-form').on('submit', function(event) {
 					event.preventDefault();
 					$('#add-user-form button[type=submit]').prop('disabled', true);
 					
-					var valid = true;
-	
+					let isValid = true;
+
 					$(this).find('input').each(function(){
-						var noValue = !$(this).val();
-						if(noValue){
-							valid = false;
+						const noValue = !$(this).val();
+						if (noValue) {
+							isValid = false;
```

Comment:
Could do something like this and get rid of the if statement:
```
isValid = !!$(this).val();
```
I'm not convinced this is better though, so it's up to you..